lms/templates/static_templates/collections.html

```diff
@@ -0,0 +1,17 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%inherit file="../main.html" />
+<%namespace name="gymcms" file="../util/gymcms.mako" />
+
+<%block name="pagetitle">${_("Collections")}</%block>
+
+<%block name="headextra">
+  ${gymcms.render('/meta/collections-meta')}
```

Comment:
May change this to `${gymcms.render('/hub-pages/meta/collections-meta')}` or whatever is appropriate, depending on the outcome of the other PR.